#goal definition
float32 target_angle
---
#result definition
float32 final_angle
---
#feedback
float32 current_angle